### PR TITLE
added various cool libs from other projects I had

### PR DESCRIPTION
**Pull request by vygandas**
Added some useful util libraries from the other stuff I had. Usually, these go from project to project so it makes sense to have them.

### CODE REVIEW

File: libs/utils/src/lib/randomString.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * @param length {number} length of generated string
+ */
+export const randomString = (length = 10) => {
+  let result = '';
+  const characters =
+    'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
```
Comment (CristoferPortela) [inline libs/utils/src/lib/randomString.ts:13]: Why not use crypto.randomBytes(64).toString('hex') ?

**Review by CristoferPortela — COMMENTED** [review]
Take a look at random string with crypto

File: libs/utils/src/lib/randomString.ts (hunk 1 above)
Comment (vygandas) [reply libs/utils/src/lib/randomString.ts:13]: I think converting to hex would limit to only abcdef, and with this approach we have full alphabet. Also, this potentially could be overriden to support custom symbols set if needed

Comment (CristoferPortela) [reply libs/utils/src/lib/randomString.ts:13]: No,  it will have many other characters, like #$)( and so on

Comment (vygandas) [reply libs/utils/src/lib/randomString.ts:13]: Ah. But what if you just want some nice confirmation code sent to email let's say. Which could look like `XCj7KO5` and would be easily writable by any user? I see your point, it's a good idea too. This functions idea is just to be able to control in full all the symbols being used.

Comment (CristoferPortela) [reply libs/utils/src/lib/randomString.ts:13]: I see, if  it must be just a confirmation code it is fine this way, I'll resolve the conversation then

**Review by CristoferPortela — APPROVED** [review]

**Pull request merged**